### PR TITLE
nginx config

### PR DESCRIPTION
**Pull request by prikshittekta**

### CODE REVIEW

File: frontend/express/public/stylesheets/main.css [hunk 1]
```diff
@@ -851,6 +851,7 @@ div.datatablesubrow {display: none;}
 .cly-multi-select.selection-exists .default-text { display: none; }
 .cly-multi-select .default-text { display: block; padding: 5px 0; }
 .cly-multi-select .select-items .item.selected { color: #2fa732; display: none; }
+.cly-multi-select.disabled { opacity: 0.4 !important; cursor: default; }
```
Comment (frknbasaran) [inline frontend/express/public/stylesheets/main.css]: is this change related with nginx config issue?

Comment (prikshittekta) [reply frontend/express/public/stylesheets/main.css]: No. its related to the multi select disabled dropdowns.
Already merged to master.
